src/components/layout-components/editor.tsx

```diff
@@ -11,17 +22,79 @@ const EditorContainer = styled(Stack, {
   position: "relative",
   padding: theme.spacing(4),
   display: "flex",
+  flexWrap: "wrap",
   flex: 1,
   flexDirection: "row",
   justifyContent: "space-between"
 }));
 
+/**
+ * Styled preview container component
+ */
+const PreviewContainer = styled(Box, {
+  label: "preview-container"
+})(({ theme }) => ({
+  borderWidth: 1,
+  borderStyle: "solid",
+  borderColor: theme.palette.primary.main,
+  display: "flex",
+  alignItems: "center",
+  justifyContent: "center",
+  width: EDITOR_SCREEN_PREVIEW_CONTAINER_WIDTH,
+  height: EDITOR_SCREEN_PREVIEW_CONTAINER_HEIGHT,
+  "&:hover": {
+    borderStyle: "solid",
+    color: theme.palette.primary.dark
+  }
+}));
+
 /**
  * Renders editor component
  */
-const Editor = () => {
+const Editor = ({ setPanelProperties }: Props) => {
+  // TODO: Waiting backend, Editor can recieve survey as props from editSurveysScreen, this will contain HTML data
+
+  const htmlTemplateDummy = titleAndTextTemplate;
+
+  const survey = {
+    pages: [
+      {
+        id: 1,
+        data: htmlTemplateDummy
+      },
+      {
+        id: 2,
+        data: htmlTemplateDummy
+      },
+      {
+        id: 3,
+        data: htmlTemplateDummy
+      }
+    ]
+  };
+
   return (
-    <EditorContainer direction="row" gap={4}>
+    <EditorContainer
+      direction="row"
+      gap={4}
+      onDoubleClick={ () => setPanelProperties(EditorPanelProperties.SURVEY) }
+    >
```

Comment:
There seems to be a lot of inconsistence between code style. In some places we've got `prop={ value }` and some other places `prop={value}`. Wasn't Rome linter/formatter used in this? Rome autoformatter should handle all of that.

Reply:
Currently we are following the industry standard, which is without gaps. The gaps should therefore only be present in older projects. Also, indeed, Rome linter has proven to be overall the best linter so far, so at least for now we should use it in new projects. It is easy to setup and it eliminates the debates over most style issues.

Reply:
The linter has not being running as it should, I will create a seperate branch with the changes from the linter (as there will be quite a few).